gcc/config/mips/psp.h

```diff
@@ -40,3 +40,7 @@ Boston, MA 02111-1307, USA.  */
 
 #undef ENDFILE_SPEC
 #define ENDFILE_SPEC "crtend%O%s crtn%O%s"
+
+/* Use int, instead of long int, for int32_t and uint32_t.  */
```

Comment:
What's the main reason for that change?

Reply:
Older versions of gcc used int instead of long int by default. The new version of gcc uses long int and prohibits mixing with int.

Reply:
But I think this is not a bug, it is expected,  we have same config in PS2.

Reply:
I don't understand why the types were changed. Developers have been using int32_t as int for many years

Reply:
Maybe @davidgfnet can bring some info here, but I also suffered that issue on the PS2 code, and I needed to change the code in some repos to adjust to current standards.

Reply:
As we agreed with @davidgfnet let's go to at lest drop this change from the PR, and let's proceed merging it

Reply:
> As we agreed with @davidgfnet let's go to at lest drop this change from the PR, and let's proceed merging it

Ok